### PR TITLE
Fixes captains hardsuit heat protection

### LINKED ISSUE #22021: Captain's hardsuit fire immunity not working
Captain's hardsuit as far as I can see is supposed to have FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT, but immediately take dammage on entering any fire.

Think this is due to missing` heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS`

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #22021

### CODE REVIEW

**Comment by KorPhaeron** [comment]
One line fix, not a balance change, can't break anything

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -718,4 +719,4 @@
 	armor = list(melee = 80, bullet = 80, laser = 50, energy = 50, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
```
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/hardsuit.dm]: You forgot to add heat protection to the helmet

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -520,6 +521,7 @@
 	allowed = list(/obj/item/weapon/tank/internals, /obj/item/device/flashlight,/obj/item/weapon/gun/energy, /obj/item/weapon/gun/ballistic, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs)
 	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
```
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/hardsuit.dm]: And now the helmet covers everything except the head

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -507,6 +507,7 @@
 	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR //we want to see the mask
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
```
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Covers body instead of head

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Wrong line, but you get the idea

**Pull request merged**